ToNote/Views/MainView.xaml

```diff
@@ -50,13 +50,35 @@
             </StackPanel>
         </DockPanel>
         <ScrollViewer HorizontalScrollBarVisibility="Auto" VerticalScrollBarVisibility="Auto" Grid.Row="1" Style="{StaticResource RightScrollViewer}">
-            <ItemsControl Margin="25" ItemsSource="{Binding Notes}">
-                <ItemsControl.ItemsPanel>
-                    <ItemsPanelTemplate>
-                        <StackPanel IsItemsHost="True"/>
-                    </ItemsPanelTemplate>
-                </ItemsControl.ItemsPanel>
-            </ItemsControl>
+            <DockPanel>
+                <StackPanel DockPanel.Dock="Top" Orientation="Horizontal">
+                    <Button Content="Select tags" Command="{Binding ToggleGroupingPanelCommand}"/>
+                    <Popup IsOpen="{Binding IsGroupingPanelOpen, Mode=TwoWay}" StaysOpen="False">
```

Comment:
This approach feels a bit hacky to me. Why didn't you use a ComboBox or something? What's the reasoning behind implementing the dropdown via a popup?

Reply:
The reasoning was provided on slack, this is more flexible. Still feels a bit hacky to me, the implementation could be smoother, but it'll do.